src/target/ext/base/singlethread.rs

```diff
@@ -32,15 +32,19 @@ pub trait SingleThreadBase: Target {
         None
     }
 
-    /// Read bytes from the specified address range and return the number of bytes that were read.
+    /// Read bytes from the specified address range and return the number of
+    /// bytes that were read.
     ///
-    /// Implementations may return a number `n` that is less than `data.len()` to indicate that
-    /// memory starting at `start_addr + n` cannot be accessed.
+    /// Implementations may return a number `n` that is less than `data.len()`
+    /// to indicate that memory starting at `start_addr + n` cannot be
+    /// accessed.
     ///
-    /// Implemenations may also return an appropriate non-fatal if the requested address range
-    /// could not be accessed (e.g: due to MMU protection, unhanded page fault, etc...).
+    /// Implemenations may also return an appropriate non-fatal if the requested
```

Comment:
missed a word. ditto in the same comment up in `multithread.rs`

```suggestion
    /// Implemenations may also return an appropriate non-fatal error if the requested
```